<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FletchDialog</class>
 <widget class="QDialog" name="FletchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure the Fletch Parameters</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Fletch Parameters</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3" >
           <widget class="QLineEdit" name="lineEditMaxIter" />
          </item>
          <item row="1" column="3" >
           <widget class="QLineEdit" name="lineEditStepSize" />
          </item>
          <item row="2" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Step Size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Precision</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Max. Iterations</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLineEdit" name="lineEditPrecision" />
          </item>
          <item row="0" column="1" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>336</width>
         <height>72</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="defaultsButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Set Defaults</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>lineEditPrecision</tabstop>
  <tabstop>lineEditStepSize</tabstop>
  <tabstop>lineEditMaxIter</tabstop>
  <tabstop>defaultsButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FletchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FletchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
